portindex2json/portindex2json.tcl

```diff
@@ -12,6 +12,43 @@
 
 package require json::write
 
+proc format_string {string} {
```

Comment:
It looks like this takes a list of strings, so the names of the proc and arg are misleading. Using `strings` rather than `string` would be better.

Reply:
Yeah, I must have been careful.  Do I include all the changes together in a separate commit or take one by one. Or rather rebase and squash with the original commits?

Sorry, but I am very new with making contributions.